shuffle/cli/src/console.rs

```diff
@@ -26,7 +26,8 @@ pub fn handle(
         import * as helpers from "{helpers}";
         import * as main from "{main}";
         import * as codegen from "{codegen}";
-        import * as help from "{repl_help}";"#,
+        import * as help from "{repl_help}";
+        import * as move from "{mv}";"#,
```

Comment:
nit: Everywhere else we import this as mv and do mv.Ascii. Let's change it for consistency? Or make everywhere move.ASCII again for consistency